src/common/snippets/include/snippets/lowered/pass/iter_handler.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "snippets/lowered/linear_ir.hpp"
+#include "snippets/lowered/pass/pass.hpp"
+#include "snippets/op/loop.hpp"
+
+namespace ov {
+namespace snippets {
+namespace lowered {
+namespace pass {
+class UpdateMemoryAccessOps : public pass::RangedPass {
```

Comment:
I'd suggest to add brief descriptions what the passes do because after first look I don't understand for what these transformations.
What do you think?

Reply:
Good idea, the description part has been added for all handlers